examples/text-replacer/popup.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--
+Copyright 2021 Google LLC
+
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file or at
+https://developers.google.com/open-source/licenses/bsd
+-->
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+  <title>Document</title>
+  <link rel="stylesheet" href="popup.css">
+</head>
+
+<body>
+  <form>
+    <table class="form--table">
+      <thead>
+        <tr>
+          <th>Find</th>
+          <th>Replace</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+          <td>
+            <input type="text" class="form--input">
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <div class="form--controls">
+      <input id="clear" type="button" value="Clear">
+      <input id="submit" type="submit" value="Replace">
+    </div>
+  </form>
+
+  <script src="popup.js"></script>
```

Comment:
Can you use `type="module"`? Then you can use top-level await in the code, rather than the awkward loading and call to `loadFormData`.